WooCommerce/Classes/Extensions/UILabel+SalesChannel.swift

```diff
@@ -23,5 +23,7 @@ private extension UILabel {
     enum Layout {
         static let borderWidth = CGFloat(0.0)
         static let cornerRadius = CGFloat(4.0)
+        static let salesChannelLabelTextColor = UIColor(named: "posPrimary")
```

Comment:
It's just my design feedback, but I think we shouldn't use secondary color on the primary one, I don't think there's enough contrast. It would make sense to me to combine `posPrimary` + `posOnPrimary` or similar.